lib/engine/game/g_1825/map.rb

```diff
@@ -544,6 +548,20 @@ def game_tiles
           append_game_tiles(gtiles, K5_TILES) if @kits[5]
           append_game_tiles(gtiles, K6_TILES) if @kits[6]
           append_game_tiles(gtiles, D1_TILES) if @optional_rules.include?(:d1)
+          gtiles = db1_tiles(gtiles) if @optional_rules.include?(:db1)
+          gtiles
+        end
+
+        def db1_tiles(gtiles)
+          gtiles.delete('87')
+          gtiles.delete('88')
+          eightysevens = ((@units[1] ? 2 : 0) + (@units[3] ? 1 : 0))
+          gtiles['887'] = eightysevens
+          gtiles['888'] = eightysevens
+          if @units[3]
+            gtiles['14'] -= 1
+            gtiles['15'] -= 1
```

Comment:
I'm sure this works, but it might be nice to generalize append_game_tiles to deal with negative counts and possibly create a "delete_game_tiles" method. You might get weird behavior if gtile[tile] was decremented to zero (i.e. it would show up in the Tiles tab).

Reply:
I'm not sure if I should read "it might be nice" as "you should do this to get the PR accepted". I would rather not if I don't have to because as I may have mentioned I am desperately cargo-culting my way through Ruby and - while I don't have a deep knowledge of many titles - I'm not aware of other cases where it would be useful and 1825's tile manifest means the zero case won't arise.

Reply:
I would strongly prefer it (Toby can make the final call). I'll just point out that having the helper methods makes adding optional rules easier.

Reply:
yep let's do @roseundy's suggestion

Reply:
OK, I'll have a crack at it tomorrow.